weave/flow/eval.py

```diff
@@ -151,7 +154,7 @@ def model_post_init(self, __context: Any) -> None:
             elif callable(scorer) and not is_op(scorer):
                 scorer = weave.op()(scorer)
```

Comment:
nit: this can be simplified to `weave.op(scorer)`

Reply:
fixed in next commit